src/style-banner.scss

```diff
@@ -169,6 +168,70 @@
   }
 }
 
+// =-=-=-=-= iconic -=-=-=-=-=-=
+
+:host([iconic]) {
+  @extend %auro-banner;
+
+  $iconBgSize: 120px;
+  .iconAbsolute {
+    position: absolute;
+  }
+  .iconContainer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    top: -$iconBgSize/6;
+  }
+
+  .roundIconBg {
+    width: $iconBgSize;
+    height: $iconBgSize;
+    background: #c4c4c4;
+    border-radius: 50%;
+    padding: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+  }
+
+  .displayImage {
+    display: none;
+    @include auro_breakpoint--sm {
```

Comment:
Where possible, group @include statements at the top of a declaration block, after any @extend statements.

https://auro.alaskaair.com/webcorestylesheets/idiomatic-css

Reply:
I do need to have display: none; before the @include statement so it hides on mobile shows on anything larger.